### PR TITLE
Feature/hit the api (Milestone #2)

### PR DESCRIPTION
**Pull request by enionsouza**
### Milestone #2
In this milestone, the following requirements have been met:
- [x] Create a new game with an arbitrary name (_My Video Game!_, in my case) by using the API;
- [x] Implement the "Refresh" button (receiving data from the API and parsing the JSON);
- [x] Implement the form "Submit" button (sending data to the API).

### CODE REVIEW

File: src/create-new-game.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/prefer-default-export */
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+async function getGameID() {
```
Comment (rloterh) [inline src/create-new-game.js]: - [x] Your `getGameID` does not follow the  `arrow function` syntax. Kindly ensure to change all methods having the keyword function to `arrow functions` so as to enhance cleaner and simpler syntax of method definition

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Eino, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦🚦

## Good Points 👍  
- API is working as expected
- No linter errors

## Required Changes ♻️
- [x] _Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/create-new-game.js (hunk 1 above)
Comment (enionsouza) [reply src/create-new-game.js]: Thanks for the tip, @rloterh! :pray:

**Review by AshakaE — APPROVED** [review]
### Hi @enionsouza  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**